build/linux/installer/conf/td-agent-bit.conf

```diff
@@ -16,7 +16,7 @@
     Tag oms.container.log.la.*
     Path ${AZMON_LOG_TAIL_PATH}
     Read_from_Head true
-    DB /var/log/omsagent-fblogs.db
+    DB /var/log/ama-logs-fblogs.db
```

Comment:
We will have to keep the same path and file names for the DB to avoid loss the DB state because of the rename.